src/main/java/EmptyDescriptionException.java

```diff
@@ -0,0 +1,21 @@
+
+class EmptyDescriptionException extends Exception {
+    private String type;
+
+    public EmptyDescriptionException(String task) {
+        super();
```

Comment:
Redundant super() call.

Reply:
Would you suggest not including the constructor method all together then? I included the super call so that I won't have an empty constructor. (If you are wondering, I am trying to avoid empty constructor so that I won't have "blanks" in my Javadoc) Thanks once again!